<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="simpanBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>SIMPAN</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="namaMemberLabel">
      <property name="text">
       <string>Nama Member</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="namaMemberLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="noTeleponLabel">
      <property name="text">
       <string>No Telepon</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="noTeleponLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="pINBBLabel">
      <property name="text">
       <string>PIN BB</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="pINBBLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="pathFotoLabel">
      <property name="text">
       <string>Path Foto</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="tanggalDaftarLabel">
      <property name="text">
       <string>Tanggal Daftar</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDateEdit" name="tanggalDaftarDateEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="pathFotoLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ubahBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hapusBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>HAPUS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cetakBtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CETAK</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>791</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NAMA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>EMAIL</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NO TLP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PIN BB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PATH FOTO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TANGGAL DAFTAR</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="cariEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
